home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/dto/CooperationDto.java

```diff
@@ -21,4 +21,6 @@
     private String usreo;
 
     private String iban;
+
+    private String adminEmail;
 }
```

Comment:
May be it will be better to name this variable headEmail or cooperationHeadEmail according to subject area terminology.